packages/sdk/src/express-authentication-middleware/express.ts

```diff
@@ -13,18 +13,25 @@ function assertAuthenticatedRequest(req: Request): asserts req is AuthenticatedR
 
   // Cast with a type assertion
   const user = req.user as Partial<{
+    app: unknown;
+    authentication: unknown;
     decodedJWT: unknown;
-    pkpAddress: unknown;
+    pkp: unknown;
     rawJWT: unknown;
   }>;
 
-  const { decodedJWT, pkpAddress, rawJWT } = user;
+  const { app, authentication, decodedJWT, pkp, rawJWT } = user;
+
+  function isDefinedObject(value: unknown): value is object {
+    return typeof value === 'object' && value !== null;
+  }
 
   if (
     typeof rawJWT !== 'string' ||
-    typeof pkpAddress !== 'string' ||
-    typeof decodedJWT !== 'object' ||
-    decodedJWT === null
+    !isDefinedObject(app) ||
```

Comment:
🤔 `pkpAddress` felt like a convenient minor duplication -- but reading this, it feels like it's getting out of hand -- maybe we should encourage devs to just reference the decoded JWT instead of copying all the data from the payload directly on to the user -- WDYT? Then they would have a consistent reference to the VincentJWT type everywhere they need data from it...